app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> int:
+    result = 0
```

Comment:
May be will be better for players type instead the  list-type to use list with instances of Players, smth like List[Players]...